cogs/displayinfo.py

```diff
@@ -0,0 +1,116 @@
+import discord
+import constantvariables
+import mongoDB
+from discord.ext import commands
+from discord.ext import tasks
+from datetime import datetime
+from datetime import date
+import datetime as dt
+from pymongo.mongo_client import MongoClient
+from pymongo.server_api import ServerApi
+import asyncio
+
+
+
+
+# Create a new client and connect to the MongoDB server
+mongoClient = MongoClient(mongoDB.MONGODB_URI, server_api=ServerApi('1'))
+
+
+try:
+    # Send a ping to confirm a successful connection to MongoDB
+    mongoClient.admin.command('ping')
+    print("Pinged your deployment. You successfully connected to MongoDB!")
+except Exception as e:
+    print(e)
+
+# Select the database
+db = mongoClient.Rasenbot
+
+
+
+class DisplayInfo(commands.Cog):
+
+    def __init__(self,client):
```

Comment:
Based on discord.py documentation, client should be the bot 
(btw, it is much more common to use `bot` over `client` now so we can look into updating those). 

Among those there's a property called `guilds`. That may be able to get us the current. We can try looking into that. 

Documentation: https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.guilds